solvebio/resource/task.py

```diff
@@ -27,3 +27,15 @@ def child_object(self):
     def follow(self):
         """ Follow the child object but do not loop """
         self.child_object.follow(loop=False)
+
+    def cancel(self):
+        """ Cancel a task """
+        if 'cancel' not in self.available_actions:
+            # It may not be queued/running or another
+            # possible reason.
+            print("Task can not be canceled.")
+            return
+
+        child_ = self.child_object
```

Comment:
Does it have to be the child object? Can't we just set cancel on the task object itself?

Reply:
Or is this what you were referring to with your comment above?